### PR TITLE
codes persist upon refresh and unique to each tab

### LINKED ISSUE #191: codes of various files are linked
## Expected Behaviour

The user should be able to write tab-specific codes

## Current Behaviour 
When users update the codes in one tab, said update occurs in other tabs

## Demo
bug

### PR DESCRIPTION
**Pull request by 7malikk**
## Description of what this Pull Request does
- [x] Codes persist upon refresh.
- [x] The user can write tab-specific codes.

## Related Issue
#180 
Closes #191 

## Recording
persists

### CODE REVIEW

**Review by DeborahIhesiaba — APPROVED** [review]

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
You haven't updated collab variable using useeffect and usestate

File: development/src/components/codingEditor.js [hunk 1]
```diff
@@ -8,15 +7,14 @@ function CodingEditor() {
...
-    '// Welcome to P2P Coder, a Community Prepared Platform...'
-  );
 
   const editorDidMount = (editor, monaco) => {
     editor.focus();
   };
 
-  const onChange = (newValue, e) => {};
+  const onChange = (newValue, e) => {
+    items.filter((e) => e.active)[0].code = newValue;
```
Comment (babblebey) [inline development/src/components/codingEditor.js:16]: This stores code state in `localStorage` item object `code` onChange to allow code state persistence. Great! 

This same function can simultaneously keep code state in a DB `document` specific to the collaboration session too at some point. I guess @Remi-dee is working on that!!

LG!

**Review by annonymousauthority — APPROVED** [review]
LG

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**